nuxeo-features/nuxeo-platform-task/nuxeo-platform-task-core/src/main/java/org/nuxeo/ecm/platform/task/core/service/TaskServiceImpl.java

```diff
@@ -603,8 +612,7 @@ public void run() {
     protected void notifyEvent(CoreSession session, Task task, List<DocumentModel> docs, String event,
             Map<String, Serializable> eventInfo, String comment, NuxeoPrincipal principal, List<String> actorIds) {
         Map<String, Serializable> eventProperties = new HashMap<>();
-        ArrayList<String> notificationRecipients = new ArrayList<>();
-        notificationRecipients.addAll(actorIds);
+        ArrayList<String> notificationRecipients = new ArrayList<>(actorIds);
```

Comment:
As you're cleaning code, could you change the declared type to `List`?

Reply:
Didn't notice. Thanks!